Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1678,6 +1687,11 @@ internal extension Purchases {
         return self.systemInfo.isSandbox
     }
 
+    /// For testing purposes
+    var isStoreKit1Configured: Bool {
+        return self.storeKitWrapper != nil
```

Comment:
starting to think we should rename this to `storeKit1Wrapper`

Reply:
Yeah that's a good idea, let me do that.

Reply:
Actually let me send a separate PR for that so we can keep the history of this particular change easier to follow in the future.